content/posts/irods-development-update-march-2023.md

```diff
@@ -0,0 +1,216 @@
+Title: iRODS Development Update: March 2023
+Date: 2023-03-28 12:00
+Author: Kory Draughn
+Slug: irods-development-update-march-2023
+Status: published
+
+
+Hello Everyone,
+
+To start things off, we've modified how resource hierarchy resolution works with replica numbers. If a client requests a specific replica by number, the resource hosting that replica will return that replica if it votes higher than 0.0. Otherwise, an error is returned. The `||` operator can be used with `DATA_RESC_HIER` in GenQuery now. `iinit` (4.3.1 only) now supports the PAM interactive authentication plugin.
+
+The iCommands have been improved. We've restored the ability to create users via `igroupadmin mkuser`. The help text for `igroupadmin` has been updated as well. Not only that, but improvements for pattern matching in `itree` have been merged and `iqstat` now supports searching all users for a specific delay rule ID.
+
+Work on groupadmin support in the Python iRODS client (PRC) is now complete. With that, raw ACL retrieval no longer fails when deleted users are encountered. Work to expose asynchronous transfer information for PUT and GET has started. The goal for this is to give applications built on top of the PRC a way to report progress.
+
+We've updated the iRODS Globus plugin to be compatible with iRODS 4.3.0. You can expect a release soon.
+
+The last item for this month is about federation and negotiation keys. If you rely on federation, then consider providing feedback on the issue at [https://github.com/irods/irods/issues/6826](https://github.com/irods/irods/issues/6826). We look forward to hearing what people from the community think about this.
```

Comment:
“people from the community think” -> “the community thinks”

Reply:
Done.